include/grpcpp/server_impl.h

```diff
@@ -199,6 +199,16 @@ class Server : public grpc::ServerInterface, private grpc::GrpcLibraryCodegen {
 
   grpc_server* server() override { return server_; }
 
+ protected:
+  void set_health_check_service(
+      std::unique_ptr<grpc::HealthCheckServiceInterface> service) {
+    health_check_service_ = std::move(service);
+  }
+
+  bool is_health_check_service_disabled() {
```

Comment:
No need for the `is_` prefix.  The style guide convention is that getters have the same name as the underlying data member.

Reply:
Where does it say that? I see in https://google.github.io/styleguide/cppguide.html#Function_Names:

"Accessors and mutators (get and set functions) may be named like variables. These often correspond to actual member variables, but this is not required. For example, int count() and void set_count(int count)."

Reply:
I think you just quoted the right text.  "These often correspond to actual member variables".

It's true that it's not absolutely required, but I believe this is a strong convention, so I think we'd need a good reason to do something different here.

Reply:
Removed the "is". I do prefer an "is" prefix for booleans, so I suppose I could rename the underlying member variable :)

Reply:
How about Lisp format? health_check_service_disabled_p . Just kidding.